layouts/index.html

```diff
@@ -106,13 +106,14 @@ <h3 class="mt-0">Encryption</h3>
               <div class="feature-extended-body">
                 <h3 class="mt-0">Organization</h3>
                 <p>
-                  Over time, you'll likely accumulate tens of entries in your
+                  Over time, you'll likely accumulate dozens of entries in your
                   vault. Aegis Authenticator has lots of organization options to
                   make finding the one you need at a particular moment easier.
-                  Set a custom icon for an entry to make it easier to find.
+                  Set a custom icon for an entry to make it easier to find it.
```

Comment:
"it" does not need to be added here; the entry is already referenced by "make _it_ easier to find".